src/doxygen.cpp

```diff
@@ -11797,7 +11797,7 @@ void searchInputFiles()
   }
   if (Doxygen::inputNameLinkedMap->empty())
   {
-    warn_uncond("No files to be processed, please check your settings, in particular INPUT, FILE_PATTERNS, and RECURSIVE");
+    warn_uncond("No files to be processed, please check your settings, in particular INPUT, FILE_PATTERNS, and RECURSIVE\n");
```

Comment:
Good catch though it would be better to have such a change not hidden in another PR but in a separate PR.

Reply:
Do you want me to create another merge request for that ?

Reply:
I think this would be better and cleaner.